.vib/kafka/runtime-parameters.yaml

```diff
@@ -2,31 +2,44 @@ externalAccess:
   enabled: true
   autoDiscovery:
     enabled: true
-  service:
-    ports:
-      client: 9092
-      controller: 9093
-      internal: 9094
-      external: 80
-    type: LoadBalancer
+  controller:
+    service:
+      ports:
+        external: 80
+      type: LoadBalancer
```

Comment:
Is this strictly necessary since Kafka is not an HTTP service? What actually happens when you try to connect to this since clients must talk to individual brokers, not through a load balancer?

Reply:
Hi @OneCricketeer,

Port 80 for external access service is not mandatory nor recommended for users.
The file `runtime-parameters.yaml` is the file used to deploy Kafka in our CI/CD while testing Kafka, and we use port 80 due to internal restrictions.